### ISSUE TITLE
Feature Request Webapp: multi autocompletion search field

### ISSUE THREAD

**Issue by Frenggi** [sbb-design-systems/design-system-webapp-documentation#65]
**Is your feature request related to a problem? Please describe.**
Currently the autocompletion can be used for a search with one single option. If you want to provide a combined search (e.g. AND-combined), for the second option no more autocompletion can be provided.

**Describe the solution you'd like**
Also for the second word (after a space) the autocompletion should be provided.
![sketch](https://user-images.githubusercontent.com/6129137/139279782-26de007d-3bed-4188-b255-5614e4dff802.png)

**Describe alternatives you've considered**
nope, not until now.

**Additional context**
The original request comes from [@daniel-sc](https://github.com/daniel-sc).

**Comment by kyubisation** [comment]
I don't quite understand the specification. Is this about an OR search for the same autocomplete?

**Comment by Frenggi** [comment]
Yes, something like that. The case we discussed for a project is, that this field is used for a search / filter. The OR search will be implemented by the application itself. But the request here would be, that for the second word (in the drawn example '123') also an autocomplete functionality is available. Today the autocomplete only searches with the complete string.
@daniel-sc: what did we discuss why a chip-input would be not adequate?

**Comment by daniel-sc** [comment]
The chip-input is normally used for selecting items from given set (e.g. labels) - we are building an "open search" (more like google).

An alternative could be to show suggestions containing the complete input (like google) - so for the input "ABCDEF 123" the second suggestion would be "ABCDEF **123**456" (or "**ABCDEF 123**456"). Not sure if this is better than the variant in the sketch.

Additionally quotemarks (`"`) should be considered to make sure the input `"a combination` (note that the user did not - yet - enter the closing quotemark) does not offer completion for `combination` but for `a combination`.

**Comment by daniel-sc** [comment]
To me this is rather a question of design/specification/concept. 
How much of the result could be archived with the current autocomplete is rather a subsequent question.

**Comment by kyubisation** [comment]
Yes, I (partially) agree.
One difference I noticed was the highlighting. Since the current implementation of the highlighting is fairly low level, I worry about changing this. Splitting the input by white space and hightlighting all parts should not be a big issue, but other variants might be more complicated.
